### ISSUE TITLE
RBM API port disableable

### ISSUE THREAD

**Issue by ParaplegicRacehorse** [ParaplegicRacehorse/RainbowMiner#1]
https://github.com/ParaplegicRacehorse/RainbowMiner/blob/docs/Docs/rainbowminer.first-run.md#if-needed-choose-a-different-api-port-default4000

**Comment by RainbowMiner** [comment]
Yes, the API port can freely be changed (and the API can be disabled, completely, if wanted).
The localhost API servers as
- convenient monitoring page on localhost:4000 and over the network machinename:4000
- if uses in "Server" mode, this will be the address and port where all clients connect to
